fetchComponent.py

```diff
@@ -65,7 +65,7 @@ def searchComponents(text, token=None, cookies=None):
 
 def getComponentInfo(lcscCode, token=None, cookies=None):
     for component in searchComponents(lcscCode, token, cookies):
-        if component["dataStr"]["head"]["c_para"]["BOM_Supplier Part"] == lcscCode:
+        if component["dataStr"]["head"]["c_para"]["Supplier Part"] == lcscCode:
```

Comment:
I can confirm this is needed, but I might change it to output something nicer if this fails again in the future.  I would probably do so by wrapping the entire thing with a `try:` / `except KeyError:` wrapper to output that the data doesn't match expected format anymore.